OpenEdXMobile/AndroidManifest.xml

```diff
@@ -43,9 +43,15 @@
             android:theme="@android:style/Theme.NoDisplay">
```

Comment:
Not needed in our case, as our SplashActivity is never in our activity stack cuz it finishes itself after redirecting to a proper activity.

Reply:
Is there any disadvantage of launching SplashActivity in singleTask mode?
I think it will be good approach to restrict the activities for multiple creation which supposed to have restriction like Splash Activity, My Courses Activity and LaunchActivity.
Thoughts ?

Reply:
When I added `singleTask` mode to the activity, it wasn't working as intended due to some reason that I didn't debug.
I just went by definition and documentation of this mode and found it unnecessary in our case.